applications/nrf5340_audio/README.rst

```diff
@@ -91,10 +86,12 @@ Connected Isochronous Stream (CIS)
   In this configuration, you can use the nRF5340 Audio development kit in the role of the gateway, the left headset, or the right headset.
 
   .. note::
-     In the current version of the nRF5340 Audio application, the CIS mode offers only monodirectional communication.
+     In the current version of the nRF5340 Audio application, the CIS mode offers both unidirectional and bidirectional communication.
```

Comment:
We need to add a section under Configuration -- "Selecting the CIS bidirectional communication" -- and describe there how to enable the bidirectional stream.

Also, we need to mention that Testing covers the unidirectional mode.

@erikrobstad , does this PR require changes in the figures? I gave a cursory look, but I'm unable to verify.

Reply:
@greg-fer It does not look like we need to change the figures

Reply:
@alexsven It does not look like you have added that (CIS) Testing covers the unidirectional mode.

Reply:
Testing as in the CI-system? The tests for bidirectional are done, just waiting for this PR to be merged

Reply:
In the documentation [Testing](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8915/nrf/applications/nrf5340_audio/README.html#id33)[](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8915/nrf/applications/nrf5340_audio/README.html#testing) section it must be mentioned that this is for the unidirectional mode.